src/action-docs.ts

```diff
@@ -15,18 +15,10 @@ export interface Options {
   includeNameHeader?: boolean;
 }
 
-interface ActionMarkdown {
-  header: string;
-  description: string;
-  inputs: string;
-  outputs: string;
-  runs: string;
-  usage: string;
-}
-
 interface ActionYml {
   name: string;
   description: string;
+  on: Record<string, WorkflowTriggerEvent>;
```

Comment:
I think it is better to define two types an use a union. The type is no called ActionYml but also valid for a workflow.

Reply:
I used one type because we need to parse the yaml and then check which elements it has to determine if it's an action or a workflow, I assume the following wouldn't work:
```
const yml = parse(readFileSync(options.actionFile, "utf-8")) as (ActionYml | WorkflowYml);
```
I can rename the type such as just `YmlStructure` to make it clearer

Reply:
Would favour to change indeed to `YmlSructure`